README.md

```diff
@@ -45,6 +45,10 @@ information, see the [Docsy documentation](https://github.com/google/docsy).
 
 To set up a local copy of the OpenCue website:
 
+1. Make sure NPM (Node Package Manager) is installed on your system.
+
+1.  Install Hugo for your system from [here](https://github.com/gohugoio/hugo/releases).
```

Comment:
Two comments about this step:

1. It might be better to point users to the guide for installing Hugo: https://gohugo.io/getting-started/installing/
2. The link text should be more descriptive. A best practice is to use the page title of the page you're linking to. For the Hugo installation instructions, you might say some something like:

Follow the steps to [install Hugo](https://gohugo.io/getting-started/installing/) for your operating system.

To learn more about choosing good link text, see https://developers.google.com/style/link-text.

Reply:
Thanks for the advice, I have followed the style and implemented.